app/bot.go

```diff
@@ -366,7 +366,7 @@ func (a *App) PermanentDeleteBot(botUserId string) *model.AppError {
 		var invErr *store.ErrInvalidInput
 		switch {
 		case errors.As(err, &invErr):
-			return model.NewAppError("PermanentDeleteBot", "app.bot.permenent_delete.bad_id", map[string]interface{}{"user_id": invErr.Value}, invErr.Error(), http.StatusBadRequest)
+			return model.NewAppError("PermanentDeleteBot", "app.bot.permanent_delete.bad_id", map[string]interface{}{"user_id": invErr.Value}, invErr.Error(), http.StatusBadRequest)
```

Comment:
It's possible these are ui facing strings in which case a number of these items would have an l10n impact...

Generally, while localizers can treat ids as opaque, it's better for them not to have typos...